src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
```

Comment:
The `href` attribute is linking to a `.scss` file. Browsers cannot directly use SASS files; you need to compile the SASS into CSS and link to the resulting `.css` file instead.